src/App.tsx

```diff
@@ -1,15 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, inputAskEdit] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
set functions should be named just like noun variable, just with "set" prefix